sdk/eventhub/event-hubs/samples-dev/iothubConnectionStringWebsockets.ts

```diff
@@ -15,7 +15,7 @@
 import * as crypto from "crypto";
 import { Buffer } from "buffer";
 import { AmqpError, Connection, ReceiverEvents, parseConnectionString } from "rhea-promise";
-import rheaPromise from "rhea-promise";
+import * as rheaPromise from "rhea-promise";
```

Comment:
looks like used only once.. couldn't you just import the only thing needed?
`isAmqpError`

Reply:
Good point! However `isAmqpError` is defined in the sample and I don't have a good name to rename either definition with so I will leave it as is for now.